websockets/websockets.go

```diff
@@ -595,6 +516,53 @@ func (w *webSocket) loop() {
 	}
 }
 
+func (w *webSocket) queueMessage(msg *message) {
+	w.tq.Queue(func() error {
+		// fmt.Println("message being processed in state", w.readyState)
+		if w.readyState != OPEN {
+			return nil // TODO maybe still emit
```

Comment:
```suggestion
			w.vu.State().Logger.Warnf("dropping message because socket is not in open state (state: %d)", w.readyState)
			return nil // TODO maybe still emit
```

How about to log this, or maybe `Debugf` it at least?

Reply:
This is behaviour change, and while there are already behaviour changes in this PR, I am not really certain of this one.

For anyone who is hitting this - this new change will basically flood their terminal. And the change we do here is very likely going to prevent any potential case where this will be considered a bug ... I think

So I will just open an issue for now